<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_faceswap">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>50</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>button/face_swap.png</normaloff>button/face_swap.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mask">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>170</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>button/mask_icon.png</normaloff>button/mask_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rabbit">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>280</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>button/rabbit.png</normaloff>button/rabbit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_nomal">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>390</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>button/nomal.png</normaloff>button/nomal.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lb_camera">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>590</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>저장하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_album">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>590</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>앨범보기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
